### PR TITLE
Add health values

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
This PR update docs to reflect current changes introduced in 
# GitHub Issues
List the GitHub issues impacted by this PR: https://github.com/dell/csi-powerflex/pull/50

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/84 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by nb950 — APPROVED** [review]

**Review by mjsdell — DISMISSED** [review]
lgtm

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -204,25 +204,29 @@ kubectl create -f deploy/kubernetes/snapshot-controller
...
 | nodeSelector             | Defines what nodes would be selected for pods of controller deployment. Leave as blank to use all nodes. Uncomment this section to deploy on master nodes exclusively.                                                                                                                                                                                                                                         | Yes     | " "     |
 | tolerations              | Defines tolerations that would be applied to controller deployment. Leave as blank to install the controller on worker nodes only. If deploying on master nodes is desired, uncomment out this section.                                                                                                                                                                                                            | Yes     | " "     |
+| **healthMonitor** |  This section configures the optional deployment of the external health monitor sidecar, for controller side volume health monitoring. | - | - |
+| enabled | Enable/disable deployment of external health monitor sidecar. | No | false |
```
Comment (AronAtDell) [inline content/docs/csidriver/installation/helm/powerflex.md:219]: "Enable/disable" should be changed to "Enable/Disable" to be consistent as below line 229

**Review by AronAtDell — APPROVED** [review]
Apart from my last comment. Everything looks good. Thanks!

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No additional edits are required.

File: content/docs/csidriver/installation/helm/powerflex.md (hunk 1 above)
Comment (JacobGros) [reply content/docs/csidriver/installation/helm/powerflex.md]: fixed

**Review by bharathsreekanth — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]
lgtm

**Pull request merged**